DatadogCore/Tests/Datadog/RUM/Integrations/CrashReportReceiverTests.swift

```diff
@@ -9,7 +9,6 @@ import TestUtilities
 import DatadogInternal
 @testable import DatadogRUM
 @testable import DatadogCrashReporting
-@testable import DatadogCore
```

Comment:
💡 It makes a significant step towards moving `CrashReportReceiverTest` to RUM tests target, instead of maintaining it in Core tests. What is left is the dependency on `DatadogCrashReporting.CrashContext` mock. This will be resolved once all CR models are moved to `DatadogInternal` 👍.